source/Octostache.Tests/VersionFixture.cs

```diff
@@ -7,7 +7,6 @@ namespace Octostache.Tests
     public class VersionFixture : BaseFixture
     {
         [Theory]
-        [InlineData("", "0", "0", "0", "0", "", "", "", "")]
```

Comment:
`Octopus.Versioning` hasn't supported empty versions since March 2021

More info (internal thread): https://octopusdeploy.slack.com/archives/C01J6U3MHJ4/p1644902367025289